source/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -77,54 +75,55 @@ Create and configure new VM
 #. Follow the *New Virtual Machine Wizard* to create a new virtual machine
    specifying the options below:
 
-   - **Name**: Choose name (e.g. ClearLinuxOS-VM).
+   - **Name**: Choose name (for example, ClearLinuxOS-VM)
```

Comment:
This looks weird to me.

Reply:
Not sure what looks weird about it? We are trying to spell out i.e. and e.g. in the docs when we see instances.

Reply:
Not sure what to say. It just does to me in a short wording/list presentation like this. 
I reserve the full words "for example" for use in a sentence, but that's just my opinion.